model_analyzer/model_manager.py

```diff
@@ -188,9 +195,26 @@ def _check_for_ensemble_model_incompatibility(
     def _init_state(self):
         """
         Sets ModelManager object managed
-        state variables in AnalyerState
+        state variables in AnalyzerState
         """
 
         self._state_manager.set_state_variable(
             'ModelManager.model_variant_name_manager',
             self._state_manager.default_encode(ModelVariantNameManager()))
+
+    def _check_for_valid_measurement(
+            self, measurement: Optional[RunConfigMeasurement]) -> None:
+        if measurement:
+            self._received_measurement_values_from_pa = True
+        else:
+            self._measurement_attempts += 1
```

Comment:
Based on the name, should this always increment?  I know you are only using it in a specific way, but we might as well do the right thing so it doesn't burn us later.  Otherwise rename to failed_measurement_attempts or something.

Reply:
Good point. I've changed variable name to `failed_measurement_attempts`.